04_verify.sh

```diff
@@ -90,12 +90,20 @@ check_k8s_entity() {
   local FAILS_CHECK="${FAILS}"
   local ENTITY
   local TYPE="${1}"
+  i=0
   shift
   for name in "${@}"; do
     # Check entity exists
     RESULT_STR="${TYPE} ${name} created"
-    ENTITY="$(kubectl --kubeconfig "${KUBECONFIG}" get "${TYPE}" "${name}" \
+    if [ "${CAPI_VERSION}" == "v1alpha3" ]; then
+      NS="$(echo "${name}" | cut -d ':' -f1)"
```

Comment:
the name and namespace are inverted in deployment vs replicaset functions. one should be changed (replica-set preferably) to have the same convention everywhere

Reply:
RS was originally 'control-plane:capbm-controller-manager', namespace was added as postfix.
This is to have check_k8s_rs() impact only in case of v1a3.
Kept as it is.